.flake8

```diff
@@ -1,2 +1,7 @@
 [flake8]
-max-line-length = 120
\ No newline at end of file
+max-line-length=120
```

Comment:
120 or 80 (PEP8) folks?

Reply:
Normally I use 100, the middle between 80 and 120, which is min and max defined by PEP8. Black default is 88. However I put 120 because it was the value on the original .flake8 file.